### PR TITLE
fix: add adityathakekar

### LINKED ISSUE #940: Adityathakekar
## Profile Screenshot

aditya-github

### PR DESCRIPTION
**Pull request by adityathakekar**
resolves #940 

## Checklist for profile addition

- [ ] Name the PR `fix: add <your-username>`.
- [ ] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [ ] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/adityathakekar.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/19209650/147535896-f3259b35-75cd-4c82-a0fb-202f3c104234.png",
+  "issueId": 940,
+  "name": "aditya thakekar",
```
Comment (syedsohan) [inline profiles/adityathakekar.json:4]:
```suggestion
  "name": "Aditya Thakekar",
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

**Review by adityathakekar — COMMENTED** [review]
Thanks syed sohan

**Review by syedsohan — APPROVED** [review]

File: profiles/adityathakekar.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/19209650/147535896-f3259b35-75cd-4c82-a0fb-202f3c104234.png",
```
Comment (krishguptadev) [inline profiles/adityathakekar.json:2]: image

```suggestion
  "image": "https://user-images.githubusercontent.com/91655303/147574750-c64d0af2-4863-4371-8a11-1f96c0531f07.png",
```

**Comment by adityathakekar** [comment]
@krishguptadev thanks man

File: profiles/adityathakekar.json (hunk 2 above)
Comment (adityathakekar) [reply profiles/adityathakekar.json:2]: 🙏Grateful

Comment (Panquesito7) [inline profiles/adityathakekar.json:2]: This was suggested by @krishguptadev. If you liked it and want to change it, you can click on the `Commit suggestion` button :) Let us know if you need any help!
```suggestion
  "image": "https://user-images.githubusercontent.com/91655303/147574750-c64d0af2-4863-4371-8a11-1f96c0531f07.png",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**